### PR TITLE
[scatter] a few more small ui tweaks

### PR DESCRIPTION
**Pull request by beets**
minor color updates, with pr/931 merged in

image

### CODE REVIEW

File: server/services/datacommons.py [hunk 1]
```diff
@@ -331,7 +331,7 @@ def get_interesting_places(dcids):
 def get_statvar_groups(dcid):
     url = API_ROOT + API_ENDPOINTS['get_statvar_groups']
     req_json = {
-        'place': dcid,
+        'places': dcid,
```
Comment (beets) [inline server/services/datacommons.py:334]: Did you run into this issue when you run your local server?

Comment (shifucun) [reply server/services/datacommons.py:334]: oh, sorry for the issue!

**Comment by beets** [comment]
merged in #931 - ptal

**Review by chejennifer — APPROVED** [review]
looks nice!

**Pull request merged**